From 9169b25304e520acc229086be6eb71a8f4b65c01 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sat, 27 Aug 2016 16:40:18 +0100
Subject: [PATCH] Don't configure for Wayland, if Wayland is not present.

---
 src/amd/vulkan/Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/amd/vulkan/Makefile.am b/src/amd/vulkan/Makefile.am
index 4b9138c..5000485 100644
--- a/src/amd/vulkan/Makefile.am
+++ b/src/amd/vulkan/Makefile.am
@@ -120,8 +120,11 @@ nodist_EXTRA_libvulkan_radeon_la_SOURCES = dummy.cpp
 libvulkan_radeon_la_SOURCES = $(VULKAN_GEM_FILES)
 
 VULKAN_ENTRYPOINT_CPPFLAGS = \
-	-DVK_USE_PLATFORM_XCB_KHR \
+	-DVK_USE_PLATFORM_XCB_KHR
+if HAVE_PLATFORM_WAYLAND
+VULKAN_ENTRYPOINT_CPPFLAGS += \
 	-DVK_USE_PLATFORM_WAYLAND_KHR
+endif
 
 radv_entrypoints.h : radv_entrypoints_gen.py $(vulkan_include_HEADERS)
 	$(AM_V_GEN)$(CPP) $(VULKAN_ENTRYPOINT_CPPFLAGS) $(top_srcdir)/include/vulkan/vulkan_intel.h |\
-- 
2.7.3

