From bf0b11a67941d6d81c3a449a68a1a52f91230309 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sun, 25 Sep 2016 17:25:16 +0100
Subject: [PATCH] Fix bo->priv_flags for 4.8.0

---
 amd/backport/kcl_ttm.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/amd/backport/kcl_ttm.c b/amd/backport/kcl_ttm.c
index 24f7a83..0676675 100644
--- a/amd/backport/kcl_ttm.c
+++ b/amd/backport/kcl_ttm.c
@@ -108,7 +108,9 @@ int _kcl_ttm_bo_init(struct ttm_bo_device *bdev,
 	bo->mem.page_alignment = page_alignment;
 	bo->mem.bus.io_reserved_vm = false;
 	bo->mem.bus.io_reserved_count = 0;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 8, 0)
 	bo->priv_flags = 0;
+#endif
 	bo->mem.placement = (TTM_PL_FLAG_SYSTEM | TTM_PL_FLAG_CACHED);
 	bo->persistent_swap_storage = persistent_swap_storage;
 	bo->acc_size = acc_size;
-- 
2.7.3

