From edc3aa0224a245f863326f4836034161c90cab6c Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sun, 25 Sep 2016 15:46:58 +0100
Subject: [PATCH 04/15] Remove extra parameter from ttm_bo_reserve for 4.7.0

---
 amd/backport/include/kcl/kcl_ttm.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/amd/backport/include/kcl/kcl_ttm.h b/amd/backport/include/kcl/kcl_ttm.h
index 3575004..4b11fb0 100644
--- a/amd/backport/include/kcl/kcl_ttm.h
+++ b/amd/backport/include/kcl/kcl_ttm.h
@@ -85,7 +85,11 @@ static inline int kcl_ttm_bo_reserve(struct ttm_buffer_object *bo,
 				 struct ww_acquire_ctx *ticket)
 {
 #if defined(BUILD_AS_DKMS)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 7, 0)
+	return ttm_bo_reserve(bo, interruptible, no_wait, ticket);
+#else
 	return ttm_bo_reserve(bo, interruptible, no_wait, false, ticket);
+#endif
 #else
 	return ttm_bo_reserve(bo, interruptible, no_wait, ticket);
 #endif
-- 
2.7.3

