From 5e55d2aeac567a10df7040da1c8e442cd7df4b76 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sun, 25 Sep 2016 16:01:43 +0100
Subject: [PATCH 07/15] Remove vblank_disable_allowed assignment

---
 amd/amdgpu/amdgpu_irq.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/amd/amdgpu/amdgpu_irq.c b/amd/amdgpu/amdgpu_irq.c
index d13865a..5a74333 100644
--- a/amd/amdgpu/amdgpu_irq.c
+++ b/amd/amdgpu/amdgpu_irq.c
@@ -240,7 +240,9 @@ int amdgpu_irq_init(struct amdgpu_device *adev)
 		INIT_WORK(&adev->hotplug_work,
 				amdgpu_hotplug_work_func);
 	}
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 7, 0)
 	adev->ddev->vblank_disable_allowed = true;
+#endif
 
 	INIT_WORK(&adev->reset_work, amdgpu_irq_reset_work_func);
 
-- 
2.7.3

