From 87495c5a9ed5e83aafd70efb8aba95f6b7682cee Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sun, 25 Sep 2016 16:38:13 +0100
Subject: [PATCH 09/15] Remove no_wait_gpu from ttm_bo_move_* functions

---
 amd/amdgpu/amdgpu_ttm.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/amd/amdgpu/amdgpu_ttm.c b/amd/amdgpu/amdgpu_ttm.c
index dbc10df..7ca1e91 100644
--- a/amd/amdgpu/amdgpu_ttm.c
+++ b/amd/amdgpu/amdgpu_ttm.c
@@ -287,8 +287,12 @@ static int amdgpu_move_blit(struct ttm_buffer_object *bo,
 			       new_mem->num_pages * PAGE_SIZE, /* bytes */
 			       bo->resv, &fence);
 	/* FIXME: handle copy error */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
+	r = ttm_bo_move_accel_cleanup(bo, fence, evict, new_mem);
+#else
 	r = ttm_bo_move_accel_cleanup(bo, fence,
 				      evict, no_wait_gpu, new_mem);
+#endif
 	fence_put(fence);
 	return r;
 }
@@ -429,7 +433,12 @@ static int amdgpu_bo_move(struct ttm_buffer_object *bo,
 
 	if (r) {
 memcpy:
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
+		r = ttm_bo_move_memcpy(bo, evict, interruptible,
+				       no_wait_gpu, new_mem);
+#else
 		r = ttm_bo_move_memcpy(bo, evict, no_wait_gpu, new_mem);
+#endif
 		if (r) {
 			return r;
 		}
-- 
2.7.3

