From 60f5ffae9e275854db04b1f63d58ed4e22e3d4df Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Sun, 25 Sep 2016 16:46:39 +0100
Subject: [PATCH 10/15] Change seq_printf format for 64 bit context

---
 amd/amdgpu/amdgpu_sa.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/amd/amdgpu/amdgpu_sa.c b/amd/amdgpu/amdgpu_sa.c
index 49d39eb..f978cbd 100644
--- a/amd/amdgpu/amdgpu_sa.c
+++ b/amd/amdgpu/amdgpu_sa.c
@@ -427,8 +427,13 @@ void amdgpu_sa_bo_dump_debug_info(struct amdgpu_sa_manager *sa_manager,
 			   soffset, eoffset, eoffset - soffset);
 
 		if (i->fence)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
+			seq_printf(m, " protected by 0x%08x on context %llu",
+				   i->fence->seqno, i->fence->context);
+#else
 			seq_printf(m, " protected by 0x%08x on context %d",
 				   i->fence->seqno, i->fence->context);
+#endif
 
 		seq_printf(m, "\n");
 	}
-- 
2.7.3

