From d202f47d8f085eeb67e88d0b41bbf8df50fb1e9e Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Mon, 4 Jul 2016 19:15:47 +0100
Subject: [PATCH 2/9] Add Gentoo as an OS option otherwise it won't build.

---
 amd/backport/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/amd/backport/Makefile b/amd/backport/Makefile
index 0c285ef..dcc94d5 100644
--- a/amd/backport/Makefile
+++ b/amd/backport/Makefile
@@ -17,8 +17,13 @@ else ifeq ("ubuntu",$(OS_NAME))
 ccflags-y += -DOS_NAME_UBUNTU
 else ifeq ("steamos",$(OS_NAME))
 ccflags-y += -DOS_NAME_STEAMOS
+else ifeq ("gentoo",$(OS_NAME))
+ccflags-y += -DOS_NAME_GENTOO
+# We don't have a version inside /etc/os-release.
+OS_VERSION = "0.0"
 else
 ccflags-y += -DOS_NAME_UNKNOWN
+OS_VERSION = "0.0"
 endif
 
 ccflags-y += \
-- 
2.7.3

