From c18d7266a50ce9353e3994ba72459e9cb29807c1 Mon Sep 17 00:00:00 2001
From: "Luke A. Guest" <laguest@archeia.com>
Date: Fri, 1 Jul 2016 12:08:49 +0100
Subject: [PATCH 6/7] Fixed location of kernel module from dkms.

---
 dkms.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dkms.conf b/dkms.conf
index 5d8209d..6878ac6 100644
--- a/dkms.conf
+++ b/dkms.conf
@@ -2,6 +2,6 @@ PACKAGE_NAME="amdgpu-pro"
 PACKAGE_VERSION="16.30.3-306809"
 BUILT_MODULE_NAME[0]="amdgpu"
 BUILT_MODULE_LOCATION[0]="amd/amdgpu"
-DEST_MODULE_LOCATION[0]="/extra"
+DEST_MODULE_LOCATION[0]="/kernel/drivers/gpu/drm/amd/amdgpu"
 AUTOINSTALL="yes"
 PRE_BUILD="pre-build.sh $kernelver"
\ No newline at end of file
-- 
2.7.3

